packages/flutter/test/material/dropdown_test.dart

```diff
@@ -380,8 +379,9 @@ void main() {
 
   testWidgets('Dropdown button with no app', (WidgetTester tester) async {
     String value = 'one';
-    void didChangeValue(String newValue) {
-      value = newValue;
+    void didChangeValue(String? newValue) {
+      if (newValue != null)
```

Comment:
and here?

Reply:
and elsewhere in this file?

Reply:
I was wondering the same by the time I got through it. Let me check again. Thanks!